templates/templated_email/account/password_reset.email

```diff
@@ -5,20 +5,20 @@
 {% endblock %}
 
 {% block plain %}
+{% trans "Hi!" context "Standard e-mail greeting" %}
 {% blocktrans context "Password reset e-mail text" %}
-Hi!
-
 You're receiving this e-mail because you or someone else has requested a password for your user account at {{ domain }}.
 It can be safely ignored if you did not request a password reset. Click the link below to reset your password.
 {% endblocktrans %}
 
 {{ reset_url }}
 
-This is an automatically generated email, please do not reply.
-
-{% blocktrans context "Password reset e-mail text" %}
-Sincerely,
-{{ site_name }}
+{% endblocktrans %}
+{% blocktrans trimmed context "Base email text" %}
+This is an automatically generated e-mail, please do not reply.
+{% endblocktrans %}
+{% blocktrans trimmed context "Base email footer" %}
+Sincerely, {{ site_name }}
```

Comment:
What about doing a `{% include ... %}`? (same for the header)

Reply:
I think it's an excellent idea.